src/main/resources/job-scheduler-config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "projectId": "client-side-events",
+  "configuration":
+  {
+    "query":
+    {
+      "writeDisposition": "WRITE_TRUNCATE",
+      "destinationTable":
+      {
+        "projectId": "client-side-events",
+        "datasetId": "Aggregate_Tables"
+      }
+    }
+  },
+
+  "every4Hours": [
```

Comment:
@ahmedalsudani I'd rather be able to specify the intervals rather than being stuck to only two.  And we could always add convenience options like "every 4 hours" if we wanted.  But I don't think we need to change anything for this pr.